csharp/nuget/NuGetPackageBuilder.ps1

```diff
@@ -0,0 +1,10 @@
+﻿$solutionRoot = Resolve-Path ..\
+
+# Delete any existing output.
+Remove-Item *.nupkg
+
+# Create new packages for any nuspec files that exist in this directory.
+Foreach ($nuspec in $(Get-Item *.nuspec))
+{
+	.\Tools\NuGet.exe pack "$nuspec"
```

Comment:
@rozele I have seen repositories commit the nuget.exe and I've seen others that do not... CIs like AppVeyor have provisions for building packages... I've also seen many flavors of the approach you mention... I would rather NOT have the nuget.exe or powershell script here so I'll remove them from the PR...